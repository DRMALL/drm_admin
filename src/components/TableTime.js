
import React from 'react'


function TableTime(str, num){
  if(typeof str === 'string')
  return (
    <div>
      <div>{ str ? str.split(' ')[0] : '' }</div>
      <div style={{ marginLeft: num }} >{ str ? str.split(' ')[1] : '' }</div>
    </div>
    )
else return
}

export default TableTime
