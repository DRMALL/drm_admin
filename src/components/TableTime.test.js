

import React from 'react'
import TableTime from './TableTime'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'

it('TableTime', ()=> {
  const wrapper = render(TableTime('2017-8-7 15:22', 15))
  expect(toJson(wrapper)).toMatchSnapshot()
})
