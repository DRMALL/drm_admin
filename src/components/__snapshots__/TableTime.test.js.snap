// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableTime 1`] = `
<div>
  <div>
    2017-8-7
  </div>
  <div
    style="margin-left:15px;"
  >
    15:22
  </div>
</div>
`;
